packages/gestalt/src/Badge.js

```diff
@@ -38,7 +55,7 @@ type Props = {|
  * ![Badge dark mode](https://raw.githubusercontent.com/pinterest/gestalt/master/playwright/visual-test/Badge-dark.spec.mjs-snapshots/Badge-dark-chromium-darwin.png)
  *
  */
-export default function Badge({ position = 'middle', text, type = 'info' }: Props): Node {
+export default function Badge({ position = 'middle', text, type = 'info', tooltip }: Props): Node {
   const TYPE_COLOR_MAP = {
```

Comment:
unrelated nit: this object should be defined outside of the function so we're not recreating it on every render

Reply:
DONE

Reply:
DONE